ufo/handlers/user.py

```diff
@@ -261,63 +274,79 @@ def user_details(user_id):
     return flask.render_template('user_details.html', user=user,
                                  invite_url=invite_url)
 
-@ufo.app.route('/user/<user_id>/delete', methods=['POST'])
+@ufo.app.route('/user/delete', methods=['POST'])
 @ufo.setup_required
-def delete_user(user_id):
+def delete_user():
   """Deletes the user corresponding to the passed in user_id from the db.
 
   If we had access to a delete method then we would not use get here.
   If the user is not found, this produces a 404 error which redirects to the
   error handler.
 
-  Args:
-    user_id: A string identifying a user in the database.
-
   Returns:
     A redirect to the user_list page after deleting the given user.
   """
+  json_id = flask.request.form.get('user_id')
+  user_id = json.loads(json_id)
```

Comment:
nit: It's actually my fault to be picky here, but I think these 2 lines would actually read better if they are folded into one line.  i.e. toss the "json_id"
```python
user_id = json.loads(flask.request.form.get('user_id'))
```